### ISSUE TITLE
ios真机闪退

### ISSUE THREAD

**Issue by cliang0917** [ivpusic/react-native-image-crop-picker#455]
### Version
Tell us which versions you are using: 

- react-native-image-crop-picker v0.16.1
- react-native v0.44.3

- iOS

模拟器没问题，打测试包装到真机后出现RN欢迎界面后闪退。

有人碰到过么。

**Comment by ivpusic** [comment]
can you please clarify the issue here/

**Comment by hellomaya** [comment]
我在模拟器遇到这个问题，后来发现，文档里面有一个 post install 的配置，就是没有配置，所以错了，里面，还有一个 release 的配置，不知道你配置了没有？就是替换原来的两个framework这样，用新的代替，你可以查看一下。

I had the similar issue at iOS simulator when running it with "react-native run-ios", and then I found that I forget the "post install" part in the doc, I also saw there is a part about "release configuration", I don't know if you have checked that? If not, please check and see if this will help, thank you.

**Comment by doyouno** [comment]
坐等解决方案。
sit still waiting for solution.
I think the library directory may be config wrong  while using pod install, thus raise up Reason : image can’t found

**Comment by hellomaya** [comment]
@doyouno 

哈哈，sit still waiting for solution, nice!! 这句翻译的好！！

**Comment by cliang0917** [comment]
谢谢。等我试试。

**Comment by ivpusic** [comment]
sorry, but I cannot read Chinese. If you still have problems, please post in plain English.

**Comment by AfirSraftGarrier** [comment]
> 谢谢。等我试试。



> 坐等解决方案。
> sit still waiting for solution.
> I think the library directory may be config wrong while using pod install, thus raise up Reason : image can’t found


https://github.com/ivpusic/react-native-image-crop-picker#post-install-steps

**Comment by AfirSraftGarrier** [comment]
答案在这：https://github.com/ivpusic/react-native-image-crop-picker#post-install-steps
